public_html/wp-content/themes/wporg-events-2023/templates/page-upcoming.html

```diff
@@ -0,0 +1,15 @@
+<!-- wp:template-part {"slug":"header"} /-->
```

Comment:
I assumed the URL would be `upcoming-events`, but this WFM too. The menu will need to be updated, though:

https://github.com/WordPress/wordcamp.org/blob/70eaf22ab3f2194a9686e15328a104fe9d21f61c/public_html/wp-content/themes/wporg-events-2023/functions.php#L38-L41

Reply:
updated to `upcoming` in 5b413d4ec2ca2d7168d758907d0ab74ea6fcccb2, 0de639a3